### DISCUSSION TITLE
Support for stylex

### DISCUSSION THREAD

**Discussion by violabg** [cschroeter/park-ui#210] [General]
Now that Meta open sourced https://stylexjs.com/ It would be nice to support it

**Comment by cschroeter** [comment]
@violabg 

Thanks for your suggestion. I've looked through the StyleX website and I could not find anything what makes this special.
Maybe you can help me understand why this is worth having

**Comment by violabg** [comment]
Well, just the fact that is an official styling system by Meta is a great news, but here is a video that may summarize better the Advantages:
https://youtu.be/v4HkkjXmEMM?si=umwRPOAfjoBM1fqI

**Comment by cschroeter** [comment]
I have some concerns about the API that Stylex has to offer. The reason being that the way I used to support other CSS Frameworks (Tailwind, Chakra) is to write an adapter function that takes a Panda Recipe and transpiles this to a Chakra theme component or a Tailwind variant. 

If the `stylex.create` fn is capable of that, then we could give it a shot. But for me priority will shift to Vue next and also establishing some sort of quality assurance for the existing frameworks. Feel free to give it a shot.
